modules/aws/vpc/outputs.tf

```diff
@@ -50,6 +50,10 @@ output "aws_lbs" {
   value = ["${compact(concat(aws_elb.api_internal.*.id, list(aws_elb.console.id), aws_elb.api_external.*.id, aws_elb.tnc.*.id))}"]
 }
 
+output "aws_lbs_length" {
```

Comment:
>This might be problematic as, anybody changing this later on will most definitely not update this in the first try...

At least it's in the same Terraform module as the logic it's replacing ;).

> Why do we need this ?

Links from 988b429 ;).

Reply:
cc @crawford WDYT?